### DISCUSSION TITLE
Cannot create layouts :(

### DISCUSSION THREAD

**Discussion by BerkeAras** [pimcore/pimcore#9174] [Q&A / Support] [answered]
Hey 👋  (_i am new to Pimcore btw._),

I just installed Pimcore X on my server and I was trying to create a new project using this document-page: https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Create_a_First_Project.html

I just created the folder structure with the files, but everytime I get this error:

`A template that extends another one cannot include content outside Twig blocks. Did you forget to put the content inside a {% block %} tag?`

<img width="1036" alt="Bildschirmfoto 2021-05-20 um 18 21 10" src="https://user-images.githubusercontent.com/36629890/119014346-2ca35000-b998-11eb-87ca-9af230fc4c06.png">

What am I doing wrong?

Thanks 🙏

**Comment by aarongerig** [comment, marked as answer]
@BerkeAras The error tells you exactly what's wrong. You can't put any content outside of a `{% block %}` tag, it has to be inside of it. You should read this documentation page: https://twig.symfony.com/doc/3.x/tags/extends.html

**Reply by BerkeAras** [reply]
Thanks for your fast answer, @aarongerig. Then, it is wrong in the documentation (pimcore)

**Reply by fashxp** [reply]
thx, updated the docs accordingly
